apps/condo/domains/notification/schema/Message.js

```diff
@@ -124,19 +137,31 @@ const Message = new GQLListSchema('Message', {
         constraints: [
             {
                 type: 'models.CheckConstraint',
-                check: 'Q(user__isnull=False) | Q(phone__isnull=False) | Q(email__isnull=False)',
-                name: 'has_phone_or_email_or_user',
+                check: 'Q(phone__isnull=False) | Q(email__isnull=False) | Q(user__isnull=False) | Q(remoteClient__isnull=False) | Q(deletedAt__isnull=False)',
+                name: 'has_phone_or_email_or_user_or_remoteClient',
             },
             {
                 type: 'models.UniqueConstraint',
                 fields: ['user', 'type', 'uniqKey'],
-                condition: 'Q(deletedAt__isnull=True)',
+                condition: 'Q(deletedAt__isnull=True) & Q(user__isnull=False) & Q(remoteClient__isnull=True)',
                 name: 'message_unique_user_type_uniqKey',
             },
+            {
+                type: 'models.UniqueConstraint',
+                fields: ['remoteClient', 'type', 'uniqKey'],
```

Comment:
We can add `['remoteClient', 'type', 'uniqKey']` without check of the user field.

Reply:
I think we don't need to add any new constraints. 

The `remoteClient` field should follow the same logic as for `email` and `phone` fields. We don't have any special constraints for that fields! 

But, the `user` field is special! Because the user can have their own notification settings. What will not be for `phone`, `email`, `remoteClient`